lib/Backends/CPU/BundleSaver.cpp

```diff
@@ -95,7 +95,7 @@ void BundleSaver::emitSymbolTable() {
   llvm::SmallVector<llvm::Constant *, 128> entries;
   // Iterate over all weights and record information about their names, offset,
   // size and kind.
-  for (auto &v : F_->getGraph()->getParent()->getConstants()) {
+  for (auto &v : F_->getGraph()->getParent()->getPlaceholders()) {
```

Comment:
I might be not up to date on this, but `Placeholders` were only for `inputs/outputs` of the network and
`Constants` are for weights.

Given that the change seems strange. Did we change the behavior for placeholders/constants?

Reply:
@rdzhabarov  We need to dump here a "symbol table" for inputs/outputs. `getConstants` was a wrong change introduced during the Placeholders refactoring.

Reply:
Yes, getConstants are not for inputs/outputs, but for static weights.